From ab8c18b7cb485ebc1c43ef7bb64a420e18016af3 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Wed, 23 Mar 2016 13:18:06 +0900
Subject: [PATCH 1420/2848] MAPREDUCE-6580. Test failure:
 TestMRJobsWithProfiler. Contributed by Eric
 Badger.

(cherry picked from commit 0bfe5a0167f0af65f1d7a8bc251a1c404ed41316)
(cherry picked from commit 12c1cc889ab955c30a8dc5ed68f1df1a5c557933)

Change-Id: Icb0ddedf121b5f13736d6c48249307b5376acf24
---
 .../mapreduce/v2/TestMRJobsWithProfiler.java       |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/v2/TestMRJobsWithProfiler.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/v2/TestMRJobsWithProfiler.java
index df55f50..cbf1f39 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/v2/TestMRJobsWithProfiler.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/v2/TestMRJobsWithProfiler.java
@@ -134,7 +134,7 @@ private void testProfilerInternal(boolean useDefault) throws Exception {
     if (!useDefault) {
       // use hprof for map to profile.out
       sleepConf.set(MRJobConfig.TASK_MAP_PROFILE_PARAMS,
-          "-agentlib:hprof=cpu=times,heap=sites,force=n,thread=y,verbose=n,"
+          "-agentlib:hprof=cpu=samples,heap=sites,force=n,thread=y,verbose=n,"
               + "file=%s");
 
       // use Xprof for reduce to stdout
-- 
1.7.9.5

